chia/daemon/server.py

```diff
@@ -244,7 +244,6 @@ async def incoming_connection(self, request):
                             self.log.error(f"Unexpected exception trying to send to websocket: {e} {tb}")
                             self.remove_connection(socket)
                             await socket.close()
-                            break
```

Comment:
Is this a regression?

Reply:
ya from the websocket move. this fixes it so less likely to close everything

Reply:
related PR https://github.com/Chia-Network/chia-blockchain/pull/12241